instat/static/InstatObject/R/Backend_Components/summary_functions.R

```diff
@@ -427,12 +427,15 @@ summary_mode <- function(x,...) {
   else return(out)
 }
 
-na_check <- function(x, na_type = "n", na_max_n = NULL, na_max_prop = NULL, na_FUN = NULL, ...) {
+na_check <- function(x, na_type = "n", na_max_n = NULL, na_max_prop = NULL, min_non_miss = NULL, na_FUN = NULL, ...) {
   if(na_type == "n") {
     return(summary_count_missing(x) <= na_max_n)
   }
   else if(na_type == "prop") {
     return(summary_count_missing(x)/summary_count(x) <= na_max_prop)
+  }
+   if(na_type == "n_non_miss") {

```

Comment:
Else If since `na_type` will only be one of these.
```suggestion
   else if(na_type == "n_non_miss") {
```

Reply:
Okay. Thanks. We'll get right to it.

Reply:
This is now ready for review.